<?xml version="1.0" encoding="UTF-8"?><sendLeadsResponse xmlns="http://soap.sforce.com/schemas/class/InchcapePublicPortalInterface_v1" xmlns:InboundIntegrationResponse="http://soap.sforce.com/schemas/class/GenericWebServiceInterface" xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><result><InboundIntegrationResponse:errorCount>1</InboundIntegrationResponse:errorCount><InboundIntegrationResponse:results><InboundIntegrationResponse:externalId>5484_2387689</InboundIntegrationResponse:externalId><InboundIntegrationResponse:objectId xsi:nil="true"/><InboundIntegrationResponse:requestId xsi:nil="true"/><InboundIntegrationResponse:status>error</InboundIntegrationResponse:status><InboundIntegrationResponse:statusMessage>The request was not accepted: duplicate value found: ExternalID__c duplicates value on record with id: a1S3E000001Ng9l</InboundIntegrationResponse:statusMessage></InboundIntegrationResponse:results><InboundIntegrationResponse:status>error</InboundIntegrationResponse:status><InboundIntegrationResponse:successCount>0</InboundIntegrationResponse:successCount></result></sendLeadsResponse>